_inc/client/components/plans/plan-price/index.jsx

```diff
@@ -51,18 +51,36 @@ export class PlanPrice extends Component {
 		const smallerPrice = this.renderPrice( priceRange[ 0 ] );
 		const higherPrice = priceRange[ 1 ] && this.renderPrice( priceRange[ 1 ] );
 
+		if ( ! higherPrice ) {
+			return (
+				<>
+					{ createInterpolateElement(
+						/* translators: This shows a price, like $22. */
+						__( '<Currency /><Price />', 'jetpack' ),
+						{
+							Currency: (
+								<sup className="plan-price__currency-symbol">{ priceRange[ 0 ].price.symbol }</sup>
+							),
+							Price: smallerPrice,
+						}
+					) }
+				</>
+			);
+		}
+
 		return (
 			<>
-				<sup className="plan-price__currency-symbol">{ priceRange[ 0 ].price.symbol }</sup>
-				{ higherPrice
-					? __( '{{smallerPrice/}}-{{higherPrice/}}', {
-							components: {
-								smallerPrice,
-								higherPrice,
-							},
-							comment: 'The price range for a particular product',
-					  } )
-					: smallerPrice }
+				{ createInterpolateElement(
+					/* translators: This shows a price range, like $ 22-55. */
+					__( '<Currency /><smallerPrice />-<higherPrice />', 'jetpack' ),
+					{
+						Currency: (
+							<sup className="plan-price__currency-symbol">{ priceRange[ 0 ].price.symbol }</sup>
+						),
+						smallerPrice: smallerPrice,
+						higherPrice: higherPrice,
+					}
+				) }
```

Comment:
I've changed the way the strings were built in that component to make them easier to translate. In some changes the currency code comes after the amount, so I think it's important to take this into account.

It's a bit hard to test since we do not have any active discounts on our plans right now. You'll need to manually edit the props on the `PlanPrice` component to be able to see both states under Jetpack > Dashboard > Plans:

<img width="697" alt="screenshot 2020-07-16 at 9 16 08" src="https://user-images.githubusercontent.com/426388/87639046-093ec180-c745-11ea-8613-a1e4d2a00621.png">
<img width="722" alt="screenshot 2020-07-16 at 9 09 38" src="https://user-images.githubusercontent.com/426388/87639051-0ba11b80-c745-11ea-9f59-ae2a4cf5053b.png">